vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSpaceController.java

```diff
@@ -53,11 +53,16 @@
     public String space(@PathVariable("id") String id, Model model) {
         ISpace space = spaceManager.getSpace(id);
         List<ISpaceLinkDisplay> spaceLinks;
+        Boolean isSpacePublished = true;
         /* (non-Javadoc)
          * Below null check is added to accommodate already existing spaces with null space status
          */
         if (space.getSpaceStatus() == null || space.getSpaceStatus().equals(SpaceStatus.PUBLISHED)
                 || authenticationFacade.getAuthenticatedUser() != null) {
+            if(space.getSpaceStatus().equals(SpaceStatus.UNPUBLISHED)) {
```

Comment:
this throws a nullpointer if `getSpaceStatus` returns null.

Reply:
In line 62 I have added a null check space.getSpaceStatus() == null. This will check if the space status is null, if it is then the admin or staff can still access the space in exhibition and therefore we must have the status isSpacePublished as false to prompt the admin/staff to publish the space.
I have verified that when we go to the exhibition directly without logging in the space throws a 404 error